src/compiler/abstract_compiler.nit

```diff
@@ -69,6 +69,8 @@ redef class ToolContext
 	var opt_no_gcc_directive = new OptionArray("Disable a advanced gcc directives for optimization", "--no-gcc-directive")
 	# --release
 	var opt_release = new OptionBool("Compile in release mode and finalize application", "--release")
+	# -g
+	var opt_debug = new OptionBool("Compile in debug mode (removes C-side optimization)", "--debug", "-g")
```

Comment:
The third person in the help message bother me. I would change the parenthesis with `(no C-side optiomization)`.